### DISCUSSION TITLE
rmse_f_trn sharply decreased to nearly zero in long trainings

### DISCUSSION THREAD

**Discussion by wxwth** [deepmodeling/deepmd-kit#3886] [Q&A]
Dear DeePMD developers,
    I am a user of DeePMD-kit (usually version 2.2.7) and DP-GEN (usually version 0.11.1). During the training of Deep Potentials, the training step was set to 400,000 steps (denoted as “short training”), and in the final iteration the training step was increased to 2,000,000 steps (denoted as “long training”).
    I found that for “long training”, the values of rmse_f_trn in the lcurve.out file sharply decreased to nearly zero after 1,000,000 steps, which occurs in all four trainings (please refer to the figure below). On the other hand, this phenomenon did not occur in a series of “short trainings”.
    I am wondering if this is a normal phenomenon, or if there are any problems in the “long training”?
    Thank you for your time and I am looking forward to your reply.
Sincerely,
Xu

![image](https://github.com/deepmodeling/deepmd-kit/assets/62468272/4e37dd75-6a6f-4938-a70e-fe8a940635ad)

[run_param_long.json.txt](https://github.com/user-attachments/files/15889635/run_param_long.json.txt)

**Comment by njzjz** [comment]
When you say "nearly zero", are they close to zero or equal to zero? Could you show the raw data?

**Reply by wxwth** [reply]
Dear Dr. Zeng,
Thank you very much for your reply! I have realized where the problem lies. Could you please delete this issue? Thank you!
